manifests/e/Eigenmiao/Rickrack/2.5.24.0/Eigenmiao.Rickrack.installer.yaml

```
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Eigenmiao.Rickrack
+PackageVersion: 2.5.24.0

```

```suggestion
PackageVersion: 2.5.24
```
This must match Control Panel.